bin/make_err

```diff
@@ -213,18 +213,98 @@ sub create_public ($) {
+    print HEADER "\n/***********************************/\n";
+    print HEADER   "/* Major error message definitions */\n";
+    print HEADER   "/***********************************/\n\n";
+    print HEADER   "/* clang-format off */\n";
+    foreach $name (sort keys %major) {
+        printf HEADER "static const H5E_msg_t ${name}_msg_s = {false, \"${major{$name}}\", H5E_MAJOR, &H5E_err_cls_s};\n";
+    }
+    print HEADER   "/* clang-format on */\n";
+
+    print_endprotect(*HEADER, $file);
+
+    # Close header file
+    close HEADER;
+}
+
+##############################################################################
+# Create the generated portion of the H5E minor message definition code
+#
+sub create_mindef ($) {
+    my $prefix = shift;         # Get the prefix for the generated file
+    my $file = "H5Emindef.h";     # Name of file to generate
+    my $name;                   # Name of error message
+    my $sect_name;              # Section of minor error messages
+
+    # Rename previous file
+#    rename "${prefix}${file}", "${prefix}${file}~" or die "unable to make backup";
+
+    # Open new header file
+    open HEADER, ">${prefix}${file}" or die "unable to modify source";
+
+    # Create file contents
+
+    print_copyright(*HEADER);
+    print_warning(*HEADER);
+    print_startprotect(*HEADER, $file);
+
+    # Iterate over all the minor error sections
+    print HEADER "\n/***********************************/\n";
+    print HEADER   "/* Minor error message definitions */\n";
+    print HEADER   "/***********************************/\n\n";
+    print HEADER   "/* clang-format off */\n";
+    foreach $sect_name (sort keys %section) {
+        print HEADER "\n/* $sect_name: $section{$sect_name} */\n";
+
+        # Iterate over all the minor errors in each section
+        for $name (sort @{$section_list{$sect_name}}) {
+            printf HEADER "static const H5E_msg_t ${name}_msg_s = {false, \"${minor{$name}}\", H5E_MINOR, &H5E_err_cls_s};\n";
+        }
+    }
+    print HEADER   "/* clang-format on */\n";
+
+    print_endprotect(*HEADER, $file);
+
+    # Close header file
+    close HEADER;
+}
+
 ##############################################################################
 # Create the generated portion of the H5E initialization code
 #
 sub create_init ($) {
     my $prefix = shift;         # Get the prefix for the generated file
     my $file = "H5Einit.h";     # Name of file to generate
     my $name;                   # Name of error message
+    my $last_name;              # Name of previous error message
     my $desc;                   # Description of error message
     my $sect_name;              # Section of minor error messages
     my $sect_desc;              # Description of section
     my $first_major = 0;        # Whether the first major error code was saved
     my $first_minor = 0;        # Whether the first minor error code was saved
+    my $num_msg;                # Number of messages
```

Comment:
I probably missed it, but as far as I can tell it looks like `num_msg` isn't used in this function. It appears to just be incremented then reset between the major and minor error sections.

Reply:
> I probably missed it, but as far as I can tell it looks like `num_msg` isn't used in this function. It appears to just be incremented then reset between the major and minor error sections.

Another leftover from copying another routine.